<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ac_mainwindow</class>
 <widget class="QWidget" name="ac_mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="ShowAc" name="ac_1" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="ShowAc" name="ac_2" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>500</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="ShowAc" name="ac_3" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>400</y>
     <width>500</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ShowAc</class>
   <extends>QWidget</extends>
   <header>showac.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
